run-cis-benchmark.sh

```diff
@@ -1,5 +1,38 @@
 #!/bin/sh
 
+test_filter() {
+
+	# -- determines if a test should be displayed or not. Returns 0 for true (should be displayed) of 1 for false (should be removed from displayed tests)
+	# $1 = type of test
+	# $2 = test serial number
+	if [[ $1 == "Scored_Only" || $1 == "Important_Scored_Only" ]]; then
+#		echo " scored only/important score if statement triggered" >&2	
```

Comment:
Can you turn these into comments that talk about the section below, or just remove them if you don't need their needed?